cdc/owner/changefeed.go

```diff
@@ -528,3 +534,20 @@ func (c *changefeed) updateStatus(currentTs int64, checkpointTs, resolvedTs mode
 func (c *changefeed) Close(ctx cdcContext.Context) {
 	c.releaseResources(ctx)
 }
+
+// addSpecialComment translate tidb feature to comment
+func addSpecialComment(ddlQuery string) (string, error) {
+	stms, _, err := parser.New().ParseSQL(ddlQuery)
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	if len(stms) != 1 {
+		log.Panic("invalid ddlQuery statement size", zap.String("ddlQuery", ddlQuery))
+	}
+	var sb strings.Builder
+	if err = stms[0].Restore(format.NewRestoreCtx(format.RestoreStringSingleQuotes|format.RestoreNameBackQuotes|
+		format.RestoreKeyWordUppercase|format.RestoreTiDBSpecialComment, &sb)); err != nil {
```

Comment:
Could you comment these flag usage correspondingly?

Also, I'd prefer the following style:

```go
// Comments ...
restoreFlags := RestoreStringSingleQuotes
// Comments ...
restoreFlags |= RestoreNameBackQuotes
...
```

Reply:
done